server/asset/eth/rpcclient.go

```diff
@@ -8,28 +8,37 @@ import (
 	"fmt"
 	"math/big"
 
+	"github.com/ethereum/go-ethereum"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/p2p"
 	"github.com/ethereum/go-ethereum/rpc"
 )
 
 // Check that rpcclient satisfies the ethFetcher interface.
 var _ ethFetcher = (*rpcclient)(nil)
 
 type rpcclient struct {
+	// ec wraps the client with some useful calls.
```

Comment:
Are you describing `ec`? Seems like you're describing the `ethclient.Client` type, which wraps an `*rpc.Client`.

Reply:
ec is a ethclient.Client. I'll reword this.